src/commands/delete-all-examples.command.ts

```diff
@@ -68,21 +68,21 @@ export class DeleteAllExamplesCommand_30 extends AbstractCommand implements ICom
         }
 
         if (this.isNullOrUndefined(this._mediaTypePath)) {
-            console.debug("[DeleteAllExamplesCommand] Could not execute the command, problem when unmashalling.");
+            console.debug("[DeleteAllExamplesCommand] Could not execute the command, problem when unmarshalling.");
             return;
         }
 
         let mediaType: Oas30MediaType = this._mediaTypePath.resolve(document) as Oas30MediaType;
 
-        if (this.isNullOrUndefined(mediaType) || this.isNullOrUndefined(mediaType.examples)) {
-            console.debug("[DeleteAllExamplesCommand] No Examples.");
+        if (this.isNullOrUndefined(mediaType)) {
+            console.debug("[DeleteAllExamplesCommand] Media type not found.");
             return;
         }
 
-        for (let k in mediaType.examples) {
+        for (let k in mediaType.getExamples()) {
             this._oldExamples[k] = this.oasLibrary().writeNode(mediaType.examples[k]);
```

Comment:
The call to `getExamples()` returns an array of `Oas30Example` objects.  So this loop isn't correct now.  Try something like this:

```typescript
mediaType.getExamples().forEach( example => {
    this._oldExamples[example.name()] = this.oasLibrary().writeNode(example);
});
```

Reply:
I've reverted that already before reading your comment:)